lib/stdlib/doc/src/gen_event.xml

```diff
@@ -845,6 +916,10 @@ gen_event:stop             ----->  Module:terminate/2
         <v>Status = term()</v>
       </type>
       <desc>
+        <warning>
+          <p>This callback is deprecated, in new code use <seemfa marker="#Module:format_status/1">
+          format_status/1</seemfa>.</p>
```

Comment:
Add a note that the this old callback function will not be called if the new one is present.